### PR TITLE
adds tips to the MOTD

### PR DESCRIPTION
**Pull request by Hughgent**
:cl: Hughgent
add: Added a tip of the round to every MOTD upon joining.
add: Fortune cookies with the wisdom of the ages.
/:cl:

[why]: Because sometimes people forget or don't know all the new features we've added.

it would look something like this.
image
it will need tips.txt in the config folder with each tip being a new line.

### CODE REVIEW

**Comment by Hughgent** [comment]
looks like this now.
image

**Comment by Ghommie** [comment]
This also could be done simiarly to /tg/, with it appearing at 30 seconds less to roundstart. 
But this way people would get tips everytime they joined, which is nice, but will probably be skipped over if the MOTD happens to be a wall of text.

**Comment by Hughgent** [comment]
image

**Comment by Hughgent** [comment]
image
image

**Comment by Rohesie** [comment]
Might want to only provide only marine-based IC-looking tips for the fortune cookies. By making more than one list.

**Comment by Hughgent** [comment]
I argue that the gods spawned only fortune cookies can afford to be "cryptic" in their messaging. 

as well as I think something special would be missed if the MOTD tips and fortune cookies didn't have the chance to be silly as well as helpful.

**Comment by Rohesie** [comment]
I'm fine with silly MOTD, but immersion should be preserved on in-character stuff.

**Comment by Hughgent** [comment]
again, the fortune cookies are spawned only via admin intervention. If a person actually bakes a fortune cookie, their inserted paper with message will overwrite anything from the list.

**Comment by Hughgent** [comment]
all tips sorted by type now.

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -1523,6 +1523,30 @@
...
+
+		H.equip_to_slot_or_del( new /obj/item/reagent_container/food/snacks/fortunecookie(H), SLOT_L_HAND )
+		if(!(istype(H.l_hand,/obj/item/reagent_container/food/snacks/fortunecookie)))
+			H.equip_to_slot_or_del( new /obj/item/reagent_container/food/snacks/fortunecookie(H), SLOT_R_HAND )
+			if(!(istype(H.r_hand,/obj/item/reagent_container/food/snacks/fortunecookie)))
+				log_admin("[key_name(H)] has their hands full, so they did not receive their fortune cookie, spawned by [key_name(src.owner)].")
+				message_admins("[key_name(H)] has their hands full, so they did not receive their fortune cookie, spawned by [key_name(src.owner)].")
+				return
+			else
+				H.update_inv_r_hand()//To ensure the icon appears in the HUD
```
Comment (Ghommie) [inline code/modules/admin/topic.dm]: This already called on `Equipped()`, and could be easily removable as far as I know.

File: code/modules/admin/topic.dm [hunk 2]
```diff
@@ -1523,6 +1523,30 @@
...
+		else
+			H.update_inv_l_hand()
```
Comment (Ghommie) [inline code/modules/admin/topic.dm]: Same.

**Comment by DominikPanic** [comment]
Conflicts.

**Comment by DominikPanic** [comment]
It may be wise to put the .txt files somewhere else as to not clutter up the config folder too much, perhaps a subfolder?

File: code/__HELPERS/global_lists.dm [hunk 3]
```diff
@@ -15,6 +15,12 @@ var/global/list/processing_machines = list()
 var/global/list/active_diseases = list()
 var/global/list/events = list()
 
+//tips
+var/global/list/marinetips = file2list('config/tips/marinetips.txt')
```
Comment (DominikPanic) [inline code/__HELPERS/global_lists.dm]: You could convert all of these into GLOB globalvars.

Comment (Hughgent) [reply code/__HELPERS/global_lists.dm]: I'll need some examples to work off of.

File: config/tips/xenotips.txt [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+As the Queen, you can force larva to grow forcibly during the first two stages, using an ability.
+As a boiler, you have the most powerful corrosive acid for melting things.
+As a boiler, your shots are more accurate when fired diagonally.
+As a boiler, shots have a small chance to bypass barricades, aim where you want them to land in case this happens.
+As a hunter, you have almost perfect stealth if you stand still.
+As a hivelord, you can upgrade existing weaker resin doors and walls by activating them in build mode via middle/shift click.
+As a praetorian, your sticky resin is vastly inferior to that of drones and hivelords.
+As a drone, your corrosive acid is very weak, but it's better than nothing.
+As a drone, spread weeds as much as you can, especially to the front line fighters.
+As a drone, you�ve got pheromones that can help the hive.
```
Comment (Rohesie) [inline config/tips/xenotips.txt]: Corrupted character. There are a few others.

File: config/tips/marinetips.txt [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+Even if a patient dies a second after being successfully revived by a defibrillator, this resets the time it takes for them to become permanently brain dead.
+Any human can make razor wire with four metal rods and barbed wire from metal sheets.
+Using a multitool on a lasgun batteries will turn it into an improvised explosive with a random timer.
+In a pinch, you can use lasgun batteries to power APCs and sentry guns.
+Lasgun batteries can be recharged in APCs and power cell rechargers if the generators are on.
+As a marine, if you are not a PFC, then you have a special room in prep with special gear vendors.
+AP bullets are not always better, they do less damage than normal but penetrate more armor.
+Did you know, that pistols are not complete garbage?
+There is no �best� loadout. Experiment and find what works for you.
+Dying is part of the game.
```
Comment (Rohesie) [inline config/tips/marinetips.txt]: This tip is meta. Please curate this section for things that would make sense IC.
Most of these tips are straight out player-to-player ones. Try to conceal them with IC parlance.

**Comment by Hughgent** [comment]
image

**Pull request merged**